packages/z/zbar/xmake.lua

```diff
@@ -0,0 +1,53 @@
+package("zbar")
+    set_homepage("https://github.com/mchehab/zbar")
+    set_description("Library for reading bar codes from various sources")
+    set_license("LGPL-2.1")
+
+    add_urls("https://github.com/mchehab/zbar/archive/refs/tags/$(version).tar.gz",
+             "https://github.com/mchehab/zbar.git")
+    add_versions("0.23.93", "212dfab527894b8bcbcc7cd1d43d63f5604a07473d31a5f02889e372614ebe28")
+
+    if is_plat("mingw") and is_subhost("msys") then
+        add_extsources("pacman::zbar")
+    elseif is_plat("linux") then
+        add_extsources("pacman::zbar", "apt::libzbar-dev")
+    elseif is_plat("macosx") then
+        add_extsources("brew::zbar")
+    end
+
+    if is_plat("linux", "bsd") then
+        add_syslinks("pthread")
+    end
+
```

Comment:
no any add_configs to set `xxx codes` options in port/xmake.lua

Reply:
Oh, we do need configs in the package description.
By the way, which form is better?
1. `xrepo install -f "qr=true" zbar`
2. `xrepo install -f "codes=qr" zbar`

For reference, the original `configure` shell script uses options like the following:
`./configure --disable-video --without-gtk --without-python --enable-shared=yes --enable-static=yes --with-pic --enable-codes=ean,databar`

Reply:
you can use single codes option/config

```lua
add_requires("xxx", configs = {codecs = {"qr", "ean", ...})
``